modules/mono/csharp_script.cpp

```diff
@@ -443,7 +443,9 @@ bool CSharpLanguage::handles_global_class_type(const String &p_type) const {
 
 String CSharpLanguage::get_global_class_name(const String &p_path, String *r_base_type, String *r_icon_path) const {
 	String class_name;
-	GDMonoCache::managed_callbacks.ScriptManagerBridge_GetGlobalClassName(&p_path, r_base_type, r_icon_path, &class_name);
+	if (gdmono && gdmono->is_runtime_initialized() && GDMonoCache::godot_api_cache_updated) {
```

Comment:
I feel like the runtime is guaranteed to be initialized at this point. Did the changes in this PR make Godot try to get the global class name of C# scripts before initialization?

Reply:
I think you are right, that was my first tentative to fix a crash in the CI pipeline with `--generate-mono-glue`. I was not able to reproduce on my computer, but since, I added a check if `ProjectSettings::get_singleton()->is_project_loaded()` is true before loading class names from disk.
I removed the check, we will see if CI pipeline passes.